src/app/developer/components/Notifications.tsx

```diff
@@ -19,7 +19,7 @@ const Notifications = ({updateAvailable, error, sendMessages}: Props) => {
   const [showError, setError] = useState(false);
 
   if (showError) {
-    throw new Error();
+    throw new Error('test error');
```

Comment:
is this `'test error'` supposed to stick around or was it a debugging thing?

Reply:
no this is irl a test error, having an actual message will help filter sentry